features/step_definitions.rb

```diff
@@ -14,24 +14,24 @@
 
 #
 
-Given %r{^I have a blank site in "(.*)"$} do |path|
-  if !File.exist?(path)
+Given(/^I have a blank site in "(.*)"$/) do |path|
```

Comment:
Which is to say, `/test/` is fine, but `/test\//` is ugly so in that case we use `%r!test/!` instead.